### PR TITLE
Split Job - Create Currency and Billing jobs

### PR DESCRIPTION
**Pull request by nreal**
# Background 
Add a new Jobs to send data of the BillinUser from Doppler database

twoJobs

Could you review it?

### CODE REVIEW

**Comment by andresmoschini** [comment]
Sorry, I am not aware of these changes and I cannot do a useful review.

File: CrossCutting/DopplerSapService/DopplerSapService.cs [hunk 1]
```diff
@@ -66,7 +66,38 @@ public async Task<HttpResponseMessage> SendCurrency(IList<CurrencyResponse> curr
...
+            };
+            var requestContent = SafeJsonConvert.SerializeObject(userBillingList, _serializationSettings);
+            httpRequest.Content = new StringContent(requestContent, Encoding.UTF8);
+            httpRequest.Content.Headers.ContentType = MediaTypeHeaderValue.Parse("application/json; charset=utf-8");
+
+            var httpResponse = new HttpResponseMessage();
+            try
+            {
+                _logger.LogInformation("Sending request to Doppler SAP Api.");
+                httpResponse = await _httpClient.SendAsync(httpRequest).ConfigureAwait(false);
```
Comment (fgchaio) [inline CrossCutting/DopplerSapService/DopplerSapService.cs:95]: I'm still thinking we should do something with this billing list. If something happens in between we won't know which data was sent to the DopplerSap service. Maybe comparing to SAP later on?
I think we should talk about this at the next meeting.

**Comment by fgchaio** [comment]
Overall looks great!! 👍

File: DopplerJobsServer/Program.cs [hunk 2]
```diff
@@ -2,7 +2,7 @@
 using Microsoft.Extensions.Hosting;
 using Serilog;
 
-namespace Doppler.Service.Job.Server
+namespace Server
```
Comment (elsupergomez) [inline DopplerJobsServer/Program.cs:5]: I think that just `Server` is very generic namespace. Could use a more specific namespace?

File: DopplerJobsServer/appsettings.json [hunk 3]
```diff
@@ -22,10 +22,14 @@
...
+    "DopplerSapJob": {
+      "IntervalCronExpression": "0 18 28 * *",
+      "Identifier": "Doppler_Billing_Job"
+    },
+    "DopplerCurrencyJob": {
       "IntervalCronExpression": "0 18 * * mon-fri",
-      "Identifier": "Doppler_Sap_Job"
-    }
+      "Identifier": "Doppler_Currency_Job"
+    }
```
Comment (elsupergomez) [inline DopplerJobsServer/appsettings.json:32]:
```suggestion
    }
```

File: DopplerJobsServer/Program.cs (hunk 2 above)
Comment (nreal) [reply DopplerJobsServer/Program.cs:5]: `Doppler.Job.Server` is more clear, I'll do

File: DopplerJobsServer/appsettings.json (hunk 3 above)
Comment (nreal) [reply DopplerJobsServer/appsettings.json]: Done

File: CrossCutting/DopplerSapService/DopplerSapService.cs [hunk 4]
```diff
@@ -66,7 +66,38 @@ public async Task<HttpResponseMessage> SendCurrency(IList<CurrencyResponse> curr
...
+            }
+            catch (Exception e)
+            {
+                _logger.LogError(e,
+                    $"Error occurred trying to send information to Doppler SAP return http code {httpResponse.StatusCode}.");
```
[the remark this answers is not in the document]
Comment (nreal) [reply CrossCutting/DopplerSapService/DopplerSapService.cs]: Done

**Pull request merged**